src/memphis.ts

```diff
@@ -864,6 +864,7 @@ class Memphis {
       let partitions = []
       try {
         createRes = this.JSONC.decode(createRes.data);
+        await this._scemaUpdatesListener(stationName, createRes.schema_update);
```

Comment:
Do you mean this producer count increase?

<img width="1236" alt="Screenshot 2023-09-14 at 6 31 13 AM" src="https://github.com/memphisdev/memphis.js/assets/30820063/1142da29-1de4-4a4c-8b7b-eb47483d8509">

Reply:
no, take a look inside the implementation of _scemaUpdatesListener function, you will find it increments some internal counter of producers

Reply:
@idanasulinmemphis I can't figure why we have the if condition there to increase the producers count.
https://github.com/memphisdev/memphis.js/blob/master/lib/memphis.js#L242

Reply:
This counter ensures that we don't create a subscription on each producer creation so wo won't end with more than 1 updates channel for the same thing.
So we can simply change the name of it to something that is more general rather than producersPerStation.
Please also make sure the other SDKs align with this fix as well